understanding/focus-not-obscured-enhanced.html

```diff
@@ -56,150 +56,109 @@
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>When a <a href="#dfn-user-interface-component">user interface component</a> receives keyboard focus, no part of the component is hidden by author-created content.
+                  <p><a href="#dfn-user-interface-component">ユーザインタフェース コンポーネント</a>がキーボードフォーカスを受け取るとき、コンテンツ制作者が作成したコンテンツによって、そのコンポーネントのどの部分も隠されることがない
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Don't cover any part of the item with focus.</dd>
+                  <dt>目標</dt>
+                  <dd>フォーカスのある項目のどの部分も覆わない。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Ensure when an item gets keyboard focus, it is fully visible.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>項目がキーボードフォーカスを取得したときに、その項目が完全に見えることを保証する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People who can't use a mouse need to see what has keyboard focus.</dd>
+                  <dt>なぜそれが重要か</dt>
+                  <dd>マウスを使用できない人々は、キーボードフォーカスがどこにあるかを確認する必要がある。</dd>
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to ensure that the item receiving keyboard
-                  focus is always visible in the user's viewport. For sighted people who rely on a keyboard
-                  (or on a device that operates through the keyboard interface, such as a switch or
-                  voice input), knowing the current point of focus is critical. The component with focus
-                  signals the interaction point on the page. Where users cannot see the item with focus,
-                  they may not know how to proceed, or may even think the system has become unresponsive.
+               <h2>意図</h2>
+               <p>この達成基準の意図は、キーボードフォーカスを受け取る項目が、利用者のビューポート内で常に表示されることを保証することである。キーボード (又はスイッチ、音声入力など、キーボードインターフェースを通じて操作するデバイス) に依存している目の見える人にとって、現在のフォーカス地点を知ることは非常に重要である。フォーカスを持つコンポーネントは、ページ上のインタラクション地点を示す。利用者がフォーカスを持つ項目を見ることができない場合、利用者はどのように進めればよいのかわからない、又はシステムが応答しなくなったとさえ思うかもしれない。
                </p>
-               <p>Typical types of content that can overlap focused items are sticky footers, sticky
-                  headers, and non-modal dialogs. As a user tabs through the page, these layers of content
-                  can hide the item receiving focus, along with its focus indicator.
+               <p>フォーカスされた項目と重なる可能性のある代表的なコンテンツの種類は、スティッキーフッター、スティッキーヘッダー、及び非モーダルダイアログである。利用者がページ内をタブで移動すると、これらのコンテンツのレイヤーにより、フォーカスを受け取った項目がフォーカスインジケータと共に覆い隠す可能性がある。
```

Comment:
「非モーダルダイアログ」ですが、「非」が「モーダル」「ダイアログ」どちらにかかっているのかが一見わかりにくい感じがしたので、「モーダル」にかかることをわかりやすくするために (つまりモードレスダイアログであることが伝わるように)、「モーダルではないダイアログ」とするのはいかがでしょうか？

```suggestion
               <p>フォーカスされた項目と重なる可能性のある代表的なコンテンツの種類は、スティッキーフッター、スティッキーヘッダー、及びモーダルではないダイアログである。利用者がページ内をタブで移動すると、これらのコンテンツのレイヤーにより、フォーカスを受け取った項目がフォーカスインジケータと共に覆い隠す可能性がある。
```